scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3937,9 +3937,19 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
       accept[Equals]
```

Comment:
There is a line above:
`val (sigName, sigTparams, sigUparamss) = if (token.is[Colon]) {`
according to spec colon must always be present.
At least one of sigName, sigTparams, sigUparamss must be present.
With this there is probably no need to fork and rollback how it was done before.

Someone noticed parsing was not straightforward and adjusted syntax? :)

Reply:
You can still have:
`given Ord[Int] with { def compare(x: Int, y: Int): Int = 0 }` 

which does not contain `:`, since `[GivenSig]` is optional.

Reply:
ahh right 👍